sky/cli.py

```diff
@@ -1519,7 +1524,7 @@ def stop(
     is_flag=True,
     required=False,
     help='Tear down the cluster instead of stopping it, when auto-stopping '
-    '(i.e., autodown rather than autostop).')
+    '(i.e., auto-down rather than auto-stop).')
```

Comment:
nit ( to make it read better): "Use auto-down (tear down the cluster; non-restartable), instead of auto-stop (restartable)."

Reply:
Done. Thanks!